### PR TITLE
cloud: add release notes for 20221227

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran.alex@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/release-notes-2022.md [hunk 1]
```diff
@@ -8,6 +8,42 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
 
+## December 27, 2022
+
+**General changes**
+
+- The point-in-time recovery (PITR) for Dedicated Tier clusters is now in General Availability (GA).
+
+    PITR supports restoring data of any point in time to a new cluster. To use the PITR feature, make sure that your TiDB cluster version is at least v6.4.0 and the TiKV node size is at least 8 vCPU and 16 GiB.
+
+    You can turn on or off the PITR feature in the Backup Settings of the [TiDB Cloud console](https://tidbcloud.com).
```
Comment (ran-huang) [inline tidb-cloud/release-notes-2022.md:19]:
```suggestion
    You can turn on or off the PITR feature in the **Backup Settings** of the [TiDB Cloud console](https://tidbcloud.com).
```

Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:19]:
```suggestion
    You can turn on or off the PITR feature in **Backup Settings** of the [TiDB Cloud console](https://tidbcloud.com).
```

File: tidb-cloud/release-notes-2022.md [hunk 2]
```diff
@@ -8,6 +8,42 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    For more information, see [Back up and restore TiDB cluster data](/tidb-cloud/backup-and-restore.md).
+
+- Support managing multiple changefeeds and editing existing changefeeds in the TiDB Cloud console.
+
+    - You can now create as many changefeeds as needed to manage different replicated data. Currently, each cluster can have up to 10 changefeeds. For more details, refer to [Changefeed overview](/tidb-cloud/changefeed-overview.md)
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:25]:
```suggestion
    - You can now create as many changefeeds as needed to manage different replicated data. Currently, each cluster can have up to 10 changefeeds. For more details, refer to [Changefeed overview](/tidb-cloud/changefeed-overview.md).
```

File: tidb-cloud/release-notes-2022.md [hunk 3]
```diff
@@ -8,6 +8,42 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+    - You can edit the configuration of an existing changefeed in the paused status. For more details, refer to [Edit a changefeed](/tidb-cloud/changefeed-overview.md#edit-a-changefeed)
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:26]:
```suggestion
    - You can edit the configuration of an existing changefeed in the paused status. For more details, refer to [Edit a changefeed](/tidb-cloud/changefeed-overview.md#edit-a-changefeed).
```

File: tidb-cloud/release-notes-2022.md [hunk 4]
```diff
@@ -8,6 +8,42 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+        - AWS Frankfurt (eu-central-1)
+    - Support multiple specifications. You can choose an appropriate specification according to the required performance to achieve optimal data migration experience.
+
+    For how to migrate data to TiDB Cloud, refer to [user documentation](/tidb-cloud/migrate-from-mysql-using-data-migration.md). For billing details, refer to [Data Migration Billing](/tidb-cloud/tidb-cloud-billing-dm.md).
+
+- Support importing local CSV files to TiDB Cloud.
+
+    It only takes a few clicks to complete the task configuration, and then your local CSV data will be quickly imported into your TiDB cluster. When using this method, you do not need to provide the cloud storage bucket path and Role ARN. The whole importing process is quick and smooth.
+
+    For more information, refer to [Import Local Files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:45]:
```suggestion
    For more information, refer to [Import local files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: tidb-cloud/release-notes-2022.md [hunk 5]
```diff
@@ -8,6 +8,42 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+- The point-in-time recovery (PITR) for Dedicated Tier clusters is now in General Availability (GA).
+
+    PITR supports restoring data of any point in time to a new cluster. To use the PITR feature, make sure that your TiDB cluster version is at least v6.4.0 and the TiKV node size is at least 8 vCPU and 16 GiB.
+
+    You can turn on or off the PITR feature in the **Backup Settings** of the [TiDB Cloud console](https://tidbcloud.com).
+
+    For more information, see [Back up and restore TiDB cluster data](/tidb-cloud/backup-and-restore.md).
+
+- Support managing multiple changefeeds and editing existing changefeeds in the TiDB Cloud console.
```
Comment (Oreoxmt) [inline tidb-cloud/release-notes-2022.md:23]:
```suggestion
- Support managing multiple changefeeds and editing existing changefeeds.
```

File: tidb-cloud/release-notes-2022.md [hunk 6]
```diff
@@ -8,6 +8,42 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+    - Provide services in the following 6 regions:
+        - AWS Oregon (us-west-2)
+        - AWS N. Virginia (us-east-1)
+        - AWS Mumbai (ap-south-1)
+        - AWS Singapore (ap-southeast-1)
+        - AWS Tokyo (ap-northeast-1)
+        - AWS Frankfurt (eu-central-1)
+    - Support multiple specifications. You can choose an appropriate specification according to the required performance to achieve optimal data migration experience.
+
+    For how to migrate data to TiDB Cloud, refer to [user documentation](/tidb-cloud/migrate-from-mysql-using-data-migration.md). For billing details, refer to [Data Migration Billing](/tidb-cloud/tidb-cloud-billing-dm.md).
```
Comment (Oreoxmt) [inline tidb-cloud/release-notes-2022.md:39]:
```suggestion
  For how to migrate data to TiDB Cloud, refer to [user documentation](/tidb-cloud/migrate-from-mysql-using-data-migration.md). For billing details, refer to [Data Migration billing](/tidb-cloud/tidb-cloud-billing-dm.md).
```

File: tidb-cloud/release-notes-2022.md [hunk 7]
```diff
@@ -8,6 +8,42 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
 
+## December 27, 2022
+
+**General changes**
+
+- The point-in-time recovery (PITR) for Dedicated Tier clusters is now in General Availability (GA).
+
+    PITR supports restoring data of any point in time to a new cluster. To use the PITR feature, make sure that your TiDB cluster version is at least v6.4.0 and the TiKV node size is at least 8 vCPU and 16 GiB.
+
+    You can turn on or off the PITR feature in the **Backup Settings** of the [TiDB Cloud console](https://tidbcloud.com).
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:19]:
```suggestion
    You can enable or disable the PITR feature in the **Backup Settings** of the [TiDB Cloud console](https://tidbcloud.com).
```

File: tidb-cloud/release-notes-2022.md [hunk 8]
```diff
@@ -8,6 +8,42 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    For more information, see [Back up and restore TiDB cluster data](/tidb-cloud/backup-and-restore.md).
+
+- Support managing multiple changefeeds and editing existing changefeeds in the TiDB Cloud console.
+
+    - You can now create as many changefeeds as needed to manage different replicated data. Currently, each cluster can have up to 10 changefeeds. For more details, refer to [Changefeed overview](/tidb-cloud/changefeed-overview.md).
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:25]:
```suggestion
    - You can now create as many changefeeds as needed to manage different data replication tasks. Currently, each cluster can have up to 10 changefeeds. For more details, refer to [Changefeed overview](/tidb-cloud/changefeed-overview.md).
```

File: tidb-cloud/release-notes-2022.md [hunk 9]
```diff
@@ -8,6 +8,42 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+    - You can edit the configuration of an existing changefeed in the paused status. For more details, refer to [Edit a changefeed](/tidb-cloud/changefeed-overview.md#edit-a-changefeed).
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:26]:
```suggestion
    - You can edit the configuration of an existing changefeed in the paused status. For more information, see [Edit a changefeed](/tidb-cloud/changefeed-overview.md#edit-a-changefeed).
```

File: tidb-cloud/release-notes-2022.md [hunk 10]
```diff
@@ -8,6 +8,42 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+        - AWS Singapore (ap-southeast-1)
+        - AWS Tokyo (ap-northeast-1)
+        - AWS Frankfurt (eu-central-1)
+    - Support multiple specifications. You can choose an appropriate specification according to the required performance to achieve optimal data migration experience.
+
+    For how to migrate data to TiDB Cloud, refer to [user documentation](/tidb-cloud/migrate-from-mysql-using-data-migration.md). For billing details, refer to [Data Migration Billing](/tidb-cloud/tidb-cloud-billing-dm.md).
+
+- Support importing local CSV files to TiDB Cloud.
+
+    It only takes a few clicks to complete the task configuration, and then your local CSV data will be quickly imported into your TiDB cluster. When using this method, you do not need to provide the cloud storage bucket path and Role ARN. The whole importing process is quick and smooth.
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:43]:
```suggestion
    It only takes a few clicks to complete the task configuration, and then your local CSV data can be quickly imported into your TiDB cluster. When using this method, you do not need to provide the cloud storage bucket path and Role ARN. The whole importing process is quick and smooth.
```

File: tidb-cloud/release-notes-2022.md [hunk 11]
```diff
@@ -8,6 +8,42 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    For more information, refer to [Import local files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:45]:
```suggestion
    For more information, see [Import local files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).
```

Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:45]:
```suggestion
    For more information, see [Import local files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).
```

Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:45]:
```suggestion
    For more information, see [Import local files to TiDB Cloud](/tidb-cloud/tidb-cloud-import-local-files.md).
```

**Review by qiancai — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Comment by baiyuqing** [comment]
Serverless upgrade TiDB Version to v6.4 today

File: tidb-cloud/release-notes-2022.md [hunk 12]
```diff
@@ -8,6 +8,42 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## December 27, 2022
+
+**General changes**
+
```
Comment (ran-huang) [inline tidb-cloud/release-notes-2022.md:14]:
```suggestion

+ Upgrade the default TiDB version of new [Serverless Tier](https://docs.pingcap.com/tidbcloud/select-cluster-tier#serverless-tier-beta) clusters from [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0) to [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0).

```

Comment (ran-huang) [reply tidb-cloud/release-notes-2022.md:14]: @baiyuqing PTAL

Comment (Oreoxmt) [reply tidb-cloud/release-notes-2022.md:14]:
```suggestion

+ Upgrade the default TiDB version of new [Serverless Tier](/select-cluster-tier.md#serverless-tier-beta) clusters from [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0) to [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0).

```
Rest LGTM

File: tidb-cloud/release-notes-2022.md [hunk 13]
```diff
@@ -8,6 +8,44 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
++ Upgrade the default TiDB version of all [Serverless Tier](/select-cluster-tier.md#serverless-tier-beta) clusters from [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0) to [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0).
```
Comment (ran-huang) [inline tidb-cloud/release-notes-2022.md:15]:
```suggestion
- Upgrade the default TiDB version of all [Serverless Tier](/select-cluster-tier.md#serverless-tier-beta) clusters from [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0) to [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0).
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: tidb-cloud/release-notes-2022.md [hunk 14]
```diff
@@ -8,6 +8,44 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## December 27, 2022
+
+**General changes**
+
+- Upgrade the default TiDB version of all [Serverless Tier](/select-cluster-tier.md#serverless-tier-beta) clusters from [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0) to [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0).
```
Comment (ran-huang) [inline tidb-cloud/release-notes-2022.md:15]:
```suggestion
- Upgrade the default TiDB version of all [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters from [v6.3.0](https://docs.pingcap.com/tidb/v6.3/release-6.3.0) to [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0).
```

**Pull request merged**